src/app/services/mail/mail.utils.ts

```diff
@@ -174,7 +149,7 @@ export const generateAutoreplyPdf = (
   return tracer.trace('generateAutoreplyPdf', () =>
```

Comment:
Now that `generatePdfFromHtml()` does its own span, we can probably remove this `span` 🤔 (because it doesn't wrap any other action than the pdf generation)

Reply:
ooh yes thanks for spotting that tim. removed! [d662ee0](https://github.com/opengovsg/FormSG/pull/5837/commits/d662ee0b245d096eb0dd00d1b8522df39a47987d)